src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -270,9 +312,9 @@ peek of what to expect in the coming months:
   platform](https://github.com/WebBluetoothCG/web-bluetooth/issues/127) will be
   added to improve user experience.
 
-At the time of writing, Chrome OS and Android 6+ are [the most advanced
+At the time of writing, Chrome OS, Android 6+ and Chrome for Linux are [the most advanced
 platforms](https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md).
-Mac OSX is partially working.  Windows 8.1+, Linux, and iOS will be supported
+Mac OSX is partially working.  Windows 8.1+ and iOS will be supported
 as much as feasible by the platforms.
```

Comment:
I'll leave it there but we might want to investigate this and add this to https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md